17/alpine/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# The MIT License
+#
+#  Copyright (c) 2015-2020, CloudBees, Inc.
```

Comment:
Can't see anything in your profile to say you work for CloudBees, probably fine to include as I assume it's 99% copied but should include others, not a lawyer though
```suggestion
#  Copyright (c) 2015-2020, CloudBees, Inc. and other Jenkins contributors
```